dags/utils/dataproc.py

```diff
@@ -178,6 +191,10 @@ def moz_dataproc_pyspark_runner(parent_dag_name=None,
                                 gcp_conn_id='google_cloud_airflow_dataproc',
                                 artifact_bucket='moz-fx-data-prod-airflow-dataproc-artifacts',
                                 storage_bucket='moz-fx-data-prod-dataproc-scratch',
+                                master_disk_type='pd-standard',
```

Comment:
Add descriptions of the new parameters to the comments starting on line 233.  Also you may want to add these parameters to the other 2 runners (jar runner, script runner)